src/plugins/workspace/public/components/workspace_overview/all_get_started_cards.ts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DEFAULT_APP_CATEGORIES } from '../../../../../core/public';
+import { GetStartCard } from './types';
+
+/**
+ * All getting start cards
+ */
+export const getStartCards: GetStartCard[] = [
```

Comment:
I guess this is only for 2.14? For 2.15, we will let plugins to register the features?

Reply:
Yes, hardcoded for 2.14.